### DISCUSSION TITLE
[question] Rates worldserver.conf

### DISCUSSION THREAD

**Discussion by xBarselonx** [azerothcore/azerothcore-wotlk#15633] [General]
Can you tell me if the extraction will change if you set it like this?
Rate.Drop.Item.Rare                     = 0,5
Rate.Drop.Item.Epic.                     = 0,5

It has to do with single-player

**Comment by BrucePrintscreen** [comment]
Hello sir,

I think you need to change the 0,5 to 0.5
Use 'point' not 'comma'
